### PR TITLE
feature/SKED-7633: Bump dependencies of skedify-sdk

### PR DESCRIPTION
**Pull request by DemianD**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = {
   globals: {
     'process.env.NODE_ENV': 'test',
   },
+  testEnvironment: 'jsdom',
```
Comment (DemianD) [inline jest.config.js:7]: https://github.com/facebook/jest/pull/9874

File: src/decorators/withResources/createRequest.js [hunk 2]
```diff
@@ -251,9 +251,8 @@ export default function createRequest(
   resourceEntity,
   { Realm, Authorization }
 ) {
-  const { instance, descriptor, resource_path, parent, requestConfig } = get(
-    resourceEntity
-  )
+  const { instance, descriptor, resource_path, parent, requestConfig } =
+    get(resourceEntity)
```
Comment (DemianD) [inline src/decorators/withResources/createRequest.js:255]: This is because of the prettier bump.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+"
 
 jobs:
-  build:
+  ci:
```
Comment (DemianD) [inline .github/workflows/ci.yml:10]: Requires a change in the protected branches

Comment (KilianLievens) [reply .github/workflows/ci.yml:10]: Done

**Review by KilianLievens — APPROVED** [review]
:clap: You are on a streak my friend.

**Pull request merged**